Userland/Libraries/LibDSP/Transport.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, kleines Filmröllchen <malu.bertsch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Music.h"
+#include <AK/Types.h>
+#include <LibCore/Object.h>
+
+namespace LibDSP {
+
+// The DAW-wide timekeeper and synchronizer
+class Transport final : public Core::Object {
+public:
+    Transport(u16 beats_per_minute, u8 beats_per_measure)
+        : m_beats_per_minute(beats_per_minute)
+        , m_beats_per_measure(beats_per_measure)
+    {
+    }
+
+    const u32& time() const { return m_time; }
```

Comment:
It doesn't make sense to return `const&` (which are basically pointers) to primitives most of the time, you should just return them by value.

Reply:
Having a reference to the time counter allows access to it without having a full transport reference, in case that's not needed. This is intentional.

Reply:
Now I understand. This changes all the time (since it's the time), right. Nevermind.